deploy/static/provider/exoscale/deploy.yaml

```diff
@@ -341,7 +341,7 @@ spec:
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx-controller
             - --election-id=ingress-controller-leader
             - --ingress-class=nginx
-            - --configmap=ingress-nginx/ingress-nginx-controller
+            - --configmap=$(POD_NAMESPACE)/ingress-nginx-controller
```

Comment:
Should not be modified here.

Reply:
I was wrong, it's just not updated before

Reply:
I was going to ask, i had not see this change before from the generate script.

Reply:
Yes, let me confirm when it was introduced